sandbox/add_version.sh

```diff
@@ -26,8 +26,13 @@ set -x
 
 mkdir "$cloud_functions_dir"
 cd "$cloud_functions_dir"
-echo "mypy==${version}
+case $version in
+  "basedmypy-"*)
+    echo "basedmypy==${version#basedmypy-}" > "requirements.in"
+  *)
```

Comment:
I see the following error when running the updated script. There might be a syntax error?
```
./sandbox/add_version.sh: line 32: syntax error near unexpected token `)'
```

Here's the shellcheck result for reference:
```
[Line 29:](javascript:setPosition(29, 1))
case $version in
^-- [SC1009](https://www.shellcheck.net/wiki/SC1009) (info): The mentioned syntax error was in this case expression.
 
[Line 30:](javascript:setPosition(30, 3))
  "basedmypy-"*)
  ^-- [SC1073](https://www.shellcheck.net/wiki/SC1073) (error): Couldn't parse this case item. Fix to allow more checks.
 
[Line 32:](javascript:setPosition(32, 4))
  *)
   ^-- [SC1072](https://www.shellcheck.net/wiki/SC1072) (error): Fix any mentioned problems and try again.
   ^-- [SC1074](https://www.shellcheck.net/wiki/SC1074) (error): Did you forget the ;; after the previous case item?
```

Reply:
ah, i see. i apologize for your wasted time here, and i appreciate the review and assistance

Reply:
It's okay. Thanks for adding a change to automate the maintenance process further :)